### PR TITLE
Migrated test_glusterd_memory_consumption_increase

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/glusterd/test_glusterd_memory_consumption_increase.py).

Updates: #292

Fixes: #687

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

File: tests/functional/glusterd/test_glusterd_memory_consumption_increase.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        # Create and start 100 volumes in a loop
+        volname = f"{self.test_name}{index}"
+        conf_dict = self.vol_type_inf[self.conv_dict['dist-rep']]
+        ret = self.redant.bulk_volume_creation(self.server_list[0], 100,
+                                               volname, conf_dict,
+                                               self.server_list,
+                                               self.brick_roots)
+
+        if not ret:
+            print("Failed to create volumes")
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_glusterd_memory_consumption_increase.py:39]: Ig it should be an exception instead :)

**Comment by nik-redhat** [comment]
@aujjwal-redhat can you try to run the test by changing the count to 10 volumes and confirm if the test is fine and add the results here.

**Comment by aujjwal-redhat** [comment]
> @aujjwal-redhat can you try to run the test by changing the count to 10 volumes and confirm if the test is fine and add the results here.

Yeah.

File: tests/functional/glusterd/test_glusterd_memory_consumption_increase.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_glusterd_memory_consumption_increase.py:39]: Oh yeah :sweat_smile:

**Comment by aujjwal-redhat** [comment]
```js
Table:
 test_glusterd_memory_consumption_increase
 
+-------------+-------------+-------------------+
| Volume Type | Test Result |  Time taken (sec) |
+-------------+-------------+-------------------+
|   Generic   |     PASS    | 617.1890532970428 |
+-------------+-------------+-------------------+



Framework runtime : 627.4527649879456

✔ Tearing down successful.
```

**Comment by nik-redhat** [comment]
@srijan-sivakumar shall we add this TC to the excluded list? It creates, starts, stops 100 volumes multiple times.

**Comment by srijan-sivakumar** [comment]
> @srijan-sivakumar shall we add this TC to the excluded list? It creates, starts, stops 100 volumes multiple times.

Yes, I think that would be good. But can we also add a comment in the yaml file stating the reason behind exclusion. That would help if someone new picks and takes a look at the yaml.

**Comment by nik-redhat** [comment]
> > @srijan-sivakumar shall we add this TC to the excluded list? It creates, starts, stops 100 volumes multiple times.
> 
> Yes, I think that would be good. But can we also add a comment in the yaml file stating the reason behind exclusion. That would help if someone new picks and takes a look at the yaml.

Cool.... @aujjwal-redhat please do the needful in this PR.

**Comment by aujjwal-redhat** [comment]
> > @srijan-sivakumar shall we add this TC to the excluded list? It creates, starts, stops 100 volumes multiple times.
> 
> Yes, I think that would be good. But can we also add a comment in the yaml file stating the reason behind exclusion. That would help if someone new picks and takes a look at the yaml.

Ok adding it in the exclude tests

**Comment by aujjwal-redhat** [comment]
@nik-redhat @srijan-sivakumar added it in excluded tests

File: config/config.yml [hunk 2]
```diff
@@ -58,3 +58,4 @@ excluded_tests:
     - tests/functional/glusterd/test_volume_set_when_glusterd_stopped_on_one_node.py
     - tests/functional/glusterd/test_gluster_volume_status_xml_dump.py
     - tests/functional/glusterd/test_peer_probe_firewall_ports_not_opened.py
+    - tests/functional/glusterd/test_glusterd_memory_consumption_increase.py
```
Comment (nik-redhat) [inline config/config.yml:61]: Do add a comment as well, specifying the reason.

Comment (aujjwal-redhat) [reply config/config.yml:61]: Aah... sorry, adding now.

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**